README.md

```diff
@@ -10,6 +10,24 @@ This Emacs package provides a minor mode which configures [Magit](https://github
   <img width=500px src="https://user-images.githubusercontent.com/52205/80056404-23745500-84f2-11ea-9ecd-832376faf2f1.png" alt="image" />
 </td></tr></table>
 
+In the event that `magit-delta-mode` is your preferred mode, one can enable this in `.emacs` like so:
+
+```emacs
+(magit-delta-mode)
+```
+
+However, this has a startup cost when executing a new `emacs` session. If `emacs` has a keybinding to start `magit-diff-buffer-file` on a buffer, this function may help if added to that keybinding:
+
+```emacs
+(setq magit-delta-loaded nil)
+(defun magit-delta-diff-buffer-file ()
+  (interactive)
+  (unless magit-delta-loaded
+      (progn
+        (magit-delta-mode)
+        (setq magit-delta-loaded t)))
+  (magit-diff-buffer-file))
+```
```

Comment:
Would adding `#'magit-delta-mode` to `magit-mode-hook` be an alternative that does basically the same thing?

Reply:
Ah, I tried searching for magit hooks, I didn't see that. Yes, this works just as well for me. It would be nice to add that to the README.md, in case anyone else doesn't know magit as well.

Reply:
Yes, I agree, let's add it to the README. Should it be like this (for `use-package` users), to avoid the possibility of toggling it off? 

```emacs-lisp
(use-package magit-delta
  :after magit
  :config
  (add-hook 'magit-mode-hook (lambda () (magit-delta-mode +1))))
```

Reply:
Awesome, let's close this pull request then. That would be much more straight forward for people to follow.